README.md

```diff
@@ -71,6 +71,23 @@ git checkout -b GH-issuenumber-title-of-issue
 
 Do you work, and commit as you see fit. Make your commit messages helpful.
 
+In general commit messages follow these guidelines 
+
+```
+A brief commit message in one or two lines
+```
+**Followed By**
+
+```
+Detailed explanation of your commit 
+
+eg : If this commit fixes a bug in the code then describe the bug, what was causing it, your solution or 
+any other relevant thing one should know if someone else works on this project.
+```
+
+You can also read articles on how to write a proper `commit message`
```

Comment:
Do you think providing a link here to the article that teaches you how to write a good commit would make sense ?

Reply:
Done, thanks